### PR TITLE
Introduce typescript to the codebase, begin porting some of the _easy_ files

### PR DESCRIPTION
**Pull request by ajpalkovic**

### CODE REVIEW

**Comment by zjrosen1** [comment]
@ajpalkovic would love to get this merged want to update with the latest develop changes and push

**Comment by ajpalkovic** [comment]
Lol i _just_ synced it up and someone pushed a conflict, sigh :(

**Comment by ajpalkovic** [comment]
There we go, CI was broken on master, but fixed it up here, so should be good to go now

File: cypress/integration/ModelPage_spec.js [hunk 1]
```diff
@@ -1,32 +1,32 @@
 describe('/state/:id', () => {
   it(`Stay At Home has the correct title & description`, () => {
```
Comment (thatneat) [inline cypress/integration/ModelPage_spec.js:2]: These expectation names are now incorrect. Probably should rename this one to match reality, and delete the other two (see comment on lines 12 and 13 where someone anticipated this  eventuality).

Ultimately it'd be nice if there was a way to write tests that didn't include the whole shebang, but probably not as part of this PR :)

```suggestion
  it(`State page the correct title & description`, () => {
```

**Pull request merged**